socorro/scripts/reprocess.py

```diff
@@ -21,7 +21,13 @@
 DESCRIPTION = """
 Sends specified crashes for reprocessing
 
-This requires SOCORRO_REPROCESS_API_TOKEN to be set in the environment to a valid API token.
+This requires SOCORRO_REPROCESS_API_TOKEN to be set in the environment to a
+valid API token.
+
+Note: If you're reprocessing more than 10,000 crashes, you should use a sleep
+value of 10 seconds. This slows down adding items to the reprocessing queue so
+they're being added roughly at the same rate as they're being processed. This
+way, the queue doesn't back up and the system doesn't start paging people.
```

Comment:
Maybe as a nice-to-have in a future PR we can avoid having to warn people about this and just do it automatically if `--sleep` isn't manually specified and there's more than 10,000 crashes?

Reply:
Yeah... so "10" is actually a fudge-factor and it's probably good enough for some/most cases, so I documented that. However, the last time I reprocessed 100,000 things, I had Miles throw in some more processors, so then the rate they're processing increased significantly and I think I was using 6 seconds as my sleep value.

I didn't want to write an essay about the calculus of appropriate sleep times, but did want to at least raise the issue and provide a "this is probably good enough for some/most cases" value.

The new infrastructure has processors that autoscale with load, so we'll want to remove this note in the future.

Reply:
Also, this is a note for you and me right now and if other developers come along, them. What language would best help us? Is something like this better?:

```
Note: If you're processing more than 10,000 crashes, you should use a sleep value
that balances the rate of crash ids being added to the queue and the rate of
crash ids being processed. Also, if you're processing a lot of crashes, you should
tell ops to increase the number of processor nodes.
```

Reply:
I like that new version except that it removed the example value. I think you're right in that adding a "good enough" value is useful, and helps orient someone unfamiliar to what scale they should use (like, don't use increments of 100+ seconds). 

Adding back the example of 10 seconds as good enough for most use cases would make this perfect.